/**:
  ros__parameters:
    class_names: ["CAR", "PEDESTRIAN", "BICYCLE"]
    point_feature_size: 4
    max_voxel_size: 40000
    point_cloud_range: [-89.6, -89.6, -3.0, 89.6, 89.6, 5.0]
    voxel_size: [0.32, 0.32, 8.0]
    downsample_factor: 1
    encoder_in_feature_size: 9
    # post-process params
    circle_nms_dist_threshold: 0.5
    iou_nms_target_class_names: ["CAR"]
    iou_nms_search_distance_2d: 10.0
    iou_nms_threshold: 0.1
    yaw_norm_thresholds: [0.3, 0.0, 0.3]
